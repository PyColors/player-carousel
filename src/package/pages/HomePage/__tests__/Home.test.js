import React from "react";

it("Should renders Home page without exploding", () => {
    const wrapper = shallow(<div>Hello Jest!</div>);
    expect(wrapper).toHaveLength(1)
    expect(wrapper).toMatchSnapshot();
});
