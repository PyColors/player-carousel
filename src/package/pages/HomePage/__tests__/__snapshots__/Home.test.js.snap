// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders Home page without exploding 1`] = `
<div>
  Hello Jest!
</div>
`;
